.github/E10-04_WARMUP_PROCESSING_SPEC.md

```
@@ -0,0 +1,215 @@
+
+    # 最大0.5秒待機
+    warmup_thread.join(OGP_WARMUP_TIMEOUT)
+  ensure
+    # 0.5秒経過してもThreadが完了していない場合、強制終了はしない
+  end
+end
+```
+
+### Thread処理の仕組み
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)